.github/copilot-instructions.md

```diff
@@ -15,171 +15,188 @@ Storybook is a large monorepo built with TypeScript, React, and various other fr
-- `sync-docs` - Documentation synchronization
-- `test-runner-build` - Test runner for built Storybook
-- `test-runner-dev` - Test runner for dev server
 - `vitest-test` - Vitest integration tests
 
-### Known Issues
-1. **Sandbox Generation Dependencies**: Sandbox creation may occasionally fail due to environment-specific issues (like Yarn version conflicts), but the GitHub API rate limiting has been resolved
-   ```bash
-   # Sandbox generation now works in CI environments
-   yarn task --task sandbox --template react-vite/default-ts
-   ```
-
-2. **Dependency Warnings**: The build process shows many peer dependency warnings, but these are expected and don't prevent successful builds
-
-3. **Large Build Times**: Most build operations take several minutes - always use appropriate timeouts
-
-4. **Sandbox Generation**: Generally reliable in CI environments, though may occasionally fail due to dependency or environment-specific issues
-
 ## Recommended Development Workflow
 
 ### For Code Changes
+
 1. Install dependencies: `yarn` (if needed)
-2. Compile packages: `yarn task --task compile`
+2. Compile packages: `yarn nx run-many -t compile -c production`
 3. Make your changes
-4. Compile packages with `cd code && yarn task --task compile`
+4. Recompile changed packages
 5. Test changes with: `cd code && yarn storybook:ui:build`
 6. Run relevant tests: `cd code && yarn test`
 
 ### For Testing UI Changes
-1. Generate a sandbox with: `yarn task --task sandbox --template [framework-template]` (may occasionally fail due to environment issues)
+
+1. Generate a sandbox: `yarn task --task sandbox --template react-vite/default-ts`
+   - Sandboxes are created at `../storybook-sandboxes/` by default
 2. If sandbox generation fails, use Storybook UI: `cd code && yarn storybook:ui`
 3. Access at http://localhost:6006/
 
 ### For Addon/Framework/Renderers Development
+
 1. Navigate to the relevant package in `code/addons/`, `code/frameworks/` or `code/renderers/`
 2. Make changes to source files
-3. Rebuild with compilation command
-4. Generate a sandbox that matches the framework/renderer being worked on
-5. Test with the appropriate test tasks
+3. Recompile: `yarn nx compile <package-name> -c production`
+4. Generate a sandbox matching the framework/renderer
+5. Test with appropriate test tasks
 
 ## Bash Command Guidelines
 
 ### Timeout Settings
+
 - **Short commands** (< 30s): Default timeout (120s) is sufficient
 - **Dependency installation**: Use 300+ seconds timeout
-- **Compilation**: Use 300+ seconds timeout  
+- **Compilation**: Use 300+ seconds timeout
 - **Linting**: Use 300+ seconds timeout
 - **Development servers**: Use async mode or timeout commands
 
 ### Example Bash Commands
+
 ```bash
 # Safe compilation with proper timeout
-bash(command="cd /path/to/storybook && yarn task --task compile", timeout=300, async=false)
+bash(command="cd /path/to/storybook && yarn nx run-many -t compile -c production", timeout=300, async=false)
```

Comment:
Generally curious why we do not always document `yarn task` and why you have decided to use the `nx` command instead?